ceph/datadog_checks/ceph/ceph.py

```diff
@@ -48,23 +48,17 @@ def _collect_raw(self, ceph_cmd, ceph_cluster, instance):
             test_sudo = os.system('setsid sudo -l < /dev/null')
             if test_sudo != 0:
                 raise Exception('The dd-agent user does not have sudo access')
-            ceph_args = ['sudo', ceph_cmd]
+            ceph_args = 'sudo {}'.format(ceph_cmd)
         else:
-            ceph_args = [ceph_cmd]
+            ceph_args = ceph_cmd
 
-        ceph_args += ['--cluster', ceph_cluster]
-
-        args = ceph_args + ['version']
-        try:
-            output, _, _ = get_subprocess_output(args, self.log)
-        except Exception as e:
-            raise Exception('Unable to run cmd=%s: %s' % (' '.join(args), str(e)))
+        ceph_args = '{} --cluster {}'.format(ceph_args, ceph_cluster)
```

Comment:
We're not running the version command anymore?

Reply:
No it didn't seem to do anything.